internal/server/v1/alert/http_handler.go

```diff
@@ -287,3 +289,37 @@ func (h *Handler) setAlertChannels(w http.ResponseWriter, r *http.Request) {
 		"alert_channels": alertChannels,
 	})
 }
+
+func (h *Handler) getAlerts(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	projectSlug := chi.URLParam(r, "project_slug")
+	resourceUrn := chi.URLParam(r, "resource_urn")
+
+	alerts, err := h.alertService.ListAlerts(ctx, projectSlug, resourceUrn)
+	if err != nil {
+		utils.WriteErr(w, err)
+		return
+	}
+
+	utils.WriteJSON(w, http.StatusOK,
+		utils.ListResponse[Alert]{Items: alerts})
+}
+
+func (h *Handler) getAlertPolicy(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	resourceType := chi.URLParam(r, "resource_type")
+	projectSlug := chi.URLParam(r, "project_slug")
+	resourceUrn := chi.URLParam(r, "resource_urn")
+
+	policy, err := h.alertService.GetAlertPolicy(ctx, projectSlug, resourceUrn, resourceType)
+	if err != nil {
+		utils.WriteErr(w, err)
+		return
+	}
+
+	var suppliedAlertVariableNames = []string{"name", "team", "entity"}
```

Comment:
we can move this outside this function, either on the top of this file or new file called `vars.go`